python/lsst/ts/MTDome/mock_llc/amcs.py

```diff
@@ -33,6 +33,8 @@
 from ..on_off import OnOff
 
 _NUM_MOTORS = 5
+_NUM_ENCODERS = 5
+_NUM_RESOLVERS = 3
```

Comment:
Good addition.

Reply:
I wonder if you should make all these public, to aid unit testing? But perhaps there are easy ways to get these values.

Reply:
I wondered about it myself as well and decided to do that once the unit tests actually use them, which is not the case at this moment. It is work in progress.